config/config_test.go

```diff
@@ -77,9 +78,21 @@ func TestConfig_ParseConfig(t *testing.T) {
 				Prefix: ReqPrefix("TEST"),
 				Level:  ReqLevel("SWH"),
 			},
-			ParentReqSpec: ReqSpec{
-				Prefix: ReqPrefix("TEST"),
-				Level:  ReqLevel("SYS"),
+			LinkSpecs: []LinkSpec{
+				{
+					Src: ReqSpec{
+						Prefix:  ReqPrefix("TEST"),
+						Level:   ReqLevel("SWH"),
+						Re:      regexp.MustCompile("REQ-TEST-SWH-(\\d+)"),
+						AttrKey: "",
+						AttrVal: regexp.MustCompile(".*")},
+					Dst: ReqSpec{
+						Prefix:  ReqPrefix("TEST"),
+						Level:   ReqLevel("SYS"),
+						Re:      regexp.MustCompile("REQ-TEST-SYS-(\\d+)"),
+						AttrKey: "",
+						AttrVal: regexp.MustCompile(".*")},
```

Comment:
This is a good example to flag the naming. In my mind Src is SYS and Dst is SWH (i.e.: Dst implements requirements from Src). But I guess this refers to the direction of the link? (from low level to high-level?)